rj_gameplay/rj_gameplay/role/unassigned_role.py

```diff
@@ -0,0 +1,17 @@
+import stp
+from rj_msgs.msg import RobotIntent
+
+
+class UnassignedRole(stp.role.Role):
+    """Role given to help role assignment correctly handle an unassigned robot"""
+
+    def __init__(self, robot: stp.rc.Robot) -> None:
+        super().__init__(robot)
+        print(f"UnassignedRole created for {robot}!")
```

Comment:
for this print, can you make it only print if the robot.visible flag is True? that way we avoid a lot of unnecessary prints that clutter the output